src/core/tests/preprocess.cpp

```diff
@@ -209,6 +212,129 @@ TEST(pre_post_process, tensor_element_type_and_scale) {
     EXPECT_EQ(ov::layout::get_layout(f->input(0)), Layout());
 }
 
+class pre_post_process_to_gray : public ::testing::TestWithParam<ColorFormat> {};
+
+TEST_P(pre_post_process_to_gray, convert_color_nhwc_to_nhwc) {
+    auto COLOR_FORMAT = GetParam();
```

Comment:
```suggestion
    const auto& COLOR_FORMAT = GetParam();
```
To avoid copy, if changed check other similar occurrences.

Reply:
Done, thanks!